app/src/main/res/values/attrs.xml

```diff
@@ -20,6 +20,9 @@
     <attr name="normalTextColor" format="color" />
     <attr name="toolbarIconColor" format="color" />
     <attr name="toolbarTextColor" format="color" />
+    <attr name="browserIconColorSelector" format="color" />
```

Comment:
this is really nice! we remove the duplicity and embrace light / dark theming at the same time!